### PR TITLE
[13.0][IMP] dms: Forgotten migration script related to https://github.com/OCA/dms/commit/0ae10d1714d806bb806c16796204e610b6c1a777 is added

### PR DESCRIPTION
**Pull request by victoralmau**
Forgotten migration script related to https://github.com/OCA/dms/commit/0ae10d1714d806bb806c16796204e610b6c1a777 is added.

It's need to apply yo 14.0 too.

Please @pedrobaeza and @Yajo  can you review it?

@Tecnativa TT30992

### CODE REVIEW

**Review by yajo — APPROVED** [review]

File: dms/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "Document Management System",
     "summary": """Document Management System for Odoo""",
-    "version": "13.0.8.0.0",
+    "version": "13.0.8.1.0",
```
Comment (dreispt) [inline dms/__manifest__.py:7]: No need to bump versions - the Bot can do that when merging

**Review by dreispt — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Should we do it for all the existing ones? Why is it needed? Isn't this computed when the field was introduced?

**Comment by victoralmau** [comment]
IMO yes, is necessary to do for all records (old records have incorrect value).
`res_mimetype` field is computed in creation and is important to preview files.

**Comment by pedrobaeza** [comment]
But why is it not recomputed when the contents change? Maybe a missing depends? Anyways, I don't see why this migration script is needed now.

**Comment by victoralmau** [comment]
Records that were created before this change have the wrong `res_mimetype` value. The field have `content` depends but it is a stored field so it will still have the wrong value.

**Comment by pedrobaeza** [comment]
OK, let's go with it although I don't fully understand it:

/ocabot merge nobump

I suppose we need it as well for v14

**Comment by victoralmau** [comment]
Finally, it's not necessary in 14.0.

File: dms/__manifest__.py (hunk 1 above)
Comment (yajo) [reply dms/__manifest__.py:7]: You need it if you want to test a migration script.

**Pull request merged**